sxg_rs/src/fetcher/js_fetcher.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+use async_trait::async_trait;
+use js_sys::Function as JsFunction;
+use wasm_bindgen::JsValue;
+use crate::http::{HttpRequest, HttpResponse};
+use super::Fetcher;
+
+pub struct JsFetcher(JsFunction);
+
```

Comment:
Ditto. Please add comments/docs for structs/methods.

Reply:
Done